db_backup/bootstrap.sh

```diff
@@ -0,0 +1,2 @@
+# assumes there is already a database called openousd
+pg_restore -U postgres --clean -d openousd /db_backup/snapshots/openousd_snapshot_2020-09-08_00-13.dump
```

Comment:
would be nice to make this a variable that we can set programmatically somewhere?